app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -2444,6 +2445,10 @@ class BrowserTabViewModel @Inject constructor(
         }
     }
 
+    fun remoteMessageId(): String {
```

Comment:
We shouldn't be exposing methods to query the state of the viewmodel from the UI. Any state should be observed and emitted through the usual flows, and if we need this info, it should be present in state/command

Reply:
+1

Reply:
Good catch! This was supposed to be temporal while implementing bradcastReceiver and I forgot to do it properly.